src/components/molecules/ArticlePagination/index.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div
+    class="pagination"
+  >
+    <app-button
+      v-if="(targetMeta.current_page === 1)"
+      small
+      hover-opacity
+      disabled
+    >
+      1
+    </app-button>
+    <app-button
+      v-else
+      small
+      hover-opacity
+      @click="paginationClick(1)"
+    >
+      1
+    </app-button>
+    <span
+      class="pagination-item-dots"
+    >
+      …
+    </span>
+    <div v-for="n in 5" :key="(n)" class="pagination-item">
```

Comment:
[must]
`()`はなくて大丈夫です！

```suggestion
    <div v-for="n in 5" :key="n" class="pagination-item">
```

Reply:
56092e8d58d5d6dab1f43ab8c4e84ad43728cd5c

()を削除しました